lib/migration/plugins/init.go

```diff
@@ -20,5 +20,5 @@ import (
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin1"
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin2"
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin5"
-	_ "github.com/vmware/vic/lib/migration/plugins/plugin7"
+	_ "github.com/vmware/vic/lib/migration/plugins/plugin6"
```

Comment:
Yes. Still getting an error. The problem is that the migrator_test needs 2 plugin versions free to update. If my plugin is last there's only one plugin version free (the MaxPluginVersion).